docs/faqs/max_execution_time.rst

```diff
@@ -7,35 +7,73 @@ What is the maximum execution time for a Qiskit Runtime job?
 To ensure fairness, there is a maximum execution time for each Qiskit Runtime job. If
 a job exceeds this time limit, it is forcibly cancelled. This is represented in the job
 status as `Cancelled - Ran too long`. The maximum execution time is the
-smaller of 1) the system limit and 2) the ``max_execution_time`` defined by the program.
-The system limit is defined as follows:
+smaller of these values: 
 
-Qiskit Runtime on IBM Cloud
----------------------------
+- The system limit 
+- The ``max_execution_time`` defined by the program
+- The ``max_execution_time`` defined by the job itself
 
-The system limit on the job execution time is 3 hours for a job running on a simulator
+The system limit depends on the channel you're using and is defined as follows:
+
+IBM Cloud system limit
+***************************
+
+The system limit on the Qiskit Runtime job execution time is 3 hours for a job running on a simulator
 and 8 hours for a job running on a physical system.
 
-Qiskit Runtime on IBM Quantum
------------------------------
+IBM Quantum system limit
+*****************************
 
-The system limit on the job execution time is
+The system limit on the Qiskit Runtime job execution time is described in the following table:
 
-+------------------+--------------+-----------+--------------+-----------+
-|                  | Public Program           | Private Program          |
-+==================+==============+===========+==============+===========+
-|                  | Premium User | Open User | Premium User | Open User |
-+------------------+--------------+-----------+--------------+-----------+
-| Simulated Device | 3h           | 1h        | 3h           |1h         |
-+------------------+--------------+-----------+--------------+-----------+
-| Real Device      | 8h           | 4h        | 8h           |2h         |
-+------------------+--------------+-----------+--------------+-----------+
++------------------+--------------+------------------+--------------+-----------+
+|                  | Primitives & prototype programs | Private programs         |
++==================+==============+==================+==============+===========+
+|                  | Premium User | Open User        | Premium User | Open User |
++------------------+--------------+------------------+--------------+-----------+
+| Simulated Device | 3h           | 1h               | 3h           | N/A       |
++------------------+--------------+------------------+--------------+-----------+
+| Real Device      | 8h           | 4h               | 8h           | N/A       |
++------------------+--------------+------------------+--------------+-----------+
 
 Note that a *premium user* here means a user who has access to backends in providers other than ibm-q/open/main.
 
+Program time limits
+***************************
+
+In addition to the ``max_execution_time`` parameter, different types of programs have their own time limits:
```

Comment:
I think we can just say "Different types of programs...", since the `max_execution_time` parameter for primitives is not visible or changeable by the end user.